lib/command/src/Obelisk/Command.hs

```diff
@@ -85,10 +85,27 @@ initSource = foldl1 (<|>)
 initForce :: Parser Bool
 initForce = switch (long "force" <> help "Allow ob init to overwrite files")
 
+
+-- | Fully qualified function path (e.g. contains module name) to the
+-- obelisk frontend function. Usually, `Frontend.frontend`. Module path is
+-- relative to the `frontend` package's source directories.
+newtype FrontendFunction = FrontendFunction
+  { unFrontendFunction :: String
+  }
+  deriving (Show, Eq, Ord)
+
+-- | Fully qualified function path (e.g. contains module name) to the
+-- obelisk backend function. For example, `Backend.backend`. The module path is
+-- relative to the `backend` package's source directories.
+newtype BackendFunction = BackendFunction
+  { unBackendFunction :: String
+  }
+  deriving (Show, Eq, Ord)
+
 data ObCommand
    = ObCommand_Init InitSource Bool
    | ObCommand_Deploy DeployCommand
-   | ObCommand_Run [(FilePath, Interpret)] (Maybe FilePath) (Maybe PortNumber)
+   | ObCommand_Run [(FilePath, Interpret)] (Maybe FilePath) (Maybe PortNumber) FrontendFunction BackendFunction
```

Comment:
Note the multiple-domains PR interferes with this by doing away with the canonical "Frontend.frontend" and making it subordinate to the domain: https://github.com/obsidiansystems/obelisk/pull/807/files#diff-9b99e714e650ab0e313f9bfe6a4f22a83697043ab6ff30c3cd27d439b85e20ecR18 

Maybe adding a constant `_backend_frontend` and keeping only `BackendFunction` makes sense here?

We temporarily did something similar for a project with https://github.com/obsidiansystems/obelisk/pull/641